testdata/Makefile

```diff
@@ -307,17 +307,17 @@ localid.db: tlspool.env
 	$(SLICOMMAND) $(CONFFILE) testcli@tlspool.arpa2.lab valexp,client,server 'tI&' /dev/null
 	$(SLICOMMAND) $(CONFFILE) testsrv@tlspool.arpa2.lab valexp,client,server 'It&' /dev/null
 	$(SLICOMMAND) $(CONFFILE) tlspool.arpa2.lab x.509,server,client '$(PRIVKEY7)' tlspool-test-webhost-cert.der
-	chown $(DMNUSR):$(DMNGRP) $(BDBENV)/* $@
+	chown $(DMNUSR):$(DMNGRP) $(BDBENV)/* $(BDBENV)/$@
 
```

Comment:
No, this is your positioning of the files.  You're welcome to do that while packaging, but not to push those distro mods back into the source :)